system/include/emscripten/em_asm.h

```diff
@@ -140,6 +141,7 @@ template<> struct __em_asm_sig<long> { static const char value = 'i'; };
 template<> struct __em_asm_sig<unsigned long> { static const char value = 'i'; };
 template<> struct __em_asm_sig<bool> { static const char value = 'i'; };
 template<> struct __em_asm_sig<wchar_t> { static const char value = 'i'; };
+template<> struct __em_asm_sig<long long> { static const char value = 'j'; };
```

Comment:
Please include a specialization for `unsigned long long` too, if possible

Reply:
Done.   However, just like with `uint32_t` these show up as signed on the JS side for now.   We can look at addressing that as a followup.